includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -67,6 +68,56 @@ private function __construct() {
 		}
 	}
 
+	/**
+	 * Handles redirect to the customizer for starter content.
+	 */
+	public function redirect_customizer() {
+		check_admin_referer( 'storefront_ecommerce_plan_starter_content' );
+
+		if ( current_user_can( 'manage_options' ) ) {
+
+			// Dismiss notice.
+			update_option( 'storefront_nux_dismissed', true );
+		}
+
+			$args = array( 'sf_starter_content' => '1' );
```

Comment:
There are a few lines like this with extra indentation. Minor, but just wanted to note this as something to keep an eye out on.